R/shared-methods.R

```diff
@@ -150,7 +91,7 @@ DataClass <- R6::R6Class(
     common_data_urls = list(),
     #' @field level_data_urls List of named lists of named links to raw data
     #' that are level specific. Any urls that share a name with a url from
-    #' `common_data_urls`. Each top level list should be named after a supported
+    #' `common_data_urls`. Each top level list should be named after a
     #' supported level.
     #' will be selected preferentially.
```

Comment:
Something got lost between these three lines, in an earlier version I see.

Reply:
:+1: